dt-impact-tracer/src/main/java/edu/washington/cs/dt/impact/plugin1/Sample.java

```diff
@@ -0,0 +1,628 @@
+                "-inputTests", dtResults + "/orig-order.txt"};
+        ImpactMain.main(args);
+
+
+
+        TestPluginPlugin.mojo().getLog().info("Cleaning Up Files");
+        try {
+            FileUtils.moveDirectory(new File(dtSubjectSource + "/sootTestOutput"), new File(dtResults + "/sootTestOutput-orig"));
+            FileUtils.deleteDirectory(new File(dtSubjectSource + "/sootOutput"));
+        } catch (Exception e){
+            e.printStackTrace();
+        }
+    }
+
+    // Setup A Subject For Test Selection
+    private void setupTestSelection(MavenProject project){
+        String JAVA_HOME = expandEnvVars("${JAVA_HOME}");
+        TestPluginPlugin.mojo().getLog().info("Test Selection: Instrumenting Test Files");
+        subprocessOutput = null;
+        try {
+            // Runtime Exec
+            String command = "java -cp " + dtTools + ":" + buildClassPath(JAVA_HOME + "/jre/lib/*") + ":" +
+                    " edu.washington.cs.dt.impact.Main.InstrumentationMain" +
+                    " -inputDir " + dtTests  +
+                    " --soot-cp " + dtLibs + ":" + dtClass + ":" + dtTests + ":" + buildClassPath(JAVA_HOME + "/jre/lib/*") +
+                    " -technique selection";
+            Process p = Runtime.getRuntime().exec(command);
+
+            // Stream Readers
+            BufferedReader stdInput = new BufferedReader(new InputStreamReader(p.getInputStream()));
+            BufferedReader stdError = new BufferedReader(new InputStreamReader(p.getErrorStream()));
+
+            // Read Command Output
+            TestPluginPlugin.mojo().getLog().info("Command Input Stream For: java -cp dtTools:JAVA_HOME/jre/lib/*:" +
+                    " edu.washington.cs.dt.impact.Main.InstrumentationMain" +
+                    " -inputDir dtTests" +
+                    " --soot-cp dtLibs:dtClass:dtTests:JAVA_HOME/jre/lib/*" +
+                    " -technique selection");
+            while ((subprocessOutput = stdInput.readLine()) != null) {
+                TestPluginPlugin.mojo().getLog().info(subprocessOutput);
+            }
+
+            // Read Command Errors
+            TestPluginPlugin.mojo().getLog().info("Command Error: Stream For: java -cp dtTools:JAVA_HOME/jre/lib/*:" +
+                    " edu.washington.cs.dt.impact.Main.InstrumentationMain" +
+                    " -inputDir dtTests" +
+                    " --soot-cp dtLibs:dtClass:dtTests:JAVA_HOME/jre/lib/*" +
+                    " -technique selection");
+            TestPluginPlugin.mojo().getLog().info(subprocessOutput);
+            while ((subprocessOutput = stdError.readLine()) != null) {
+                TestPluginPlugin.mojo().getLog().error(subprocessOutput);
+            }
+        }
+        catch (IOException e) {
+            e.printStackTrace();
+        }
+
+
+
+        TestPluginPlugin.mojo().getLog().info("Test Selection: Instrumenting Source Files");
+        subprocessOutput = null;
+        try {
```

Comment:
Minor: This pattern (run command, print output) is repeated a lot, should probably be refactored into a method.